__tests__/app.test.js

```diff
@@ -25,4 +25,22 @@ describe("/api/topics", () => {
             })
         })
     })
+})
+
+describe("/api", () => {
+    test("Get: 200 sends an object of all available endpoints", () => {
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then((response) => {
+            expect(response.body["GET /api"]).toEqual({"description": "serves up a json representation of all the available endpoints of the api"})
+            expect(response.body["GET /api/topics"]).toEqual({
+                "description": "serves an array of all topics",
+                "queries": [],
+                "exampleResponse": {
+                  "topics": [{ "slug": "football", "description": "Footie!" }]
+                }
```

Comment:
Solid test, could import the endpoints.json file to the test file to save you typing this out for every endpoint!